dapps/profile.vue/src/components/profile/profile.scss

```diff
@@ -21,4 +21,26 @@
   .profile-verification-card {
     margin-top: 1em;
   }
+
+  .row {
```

Comment:
I think creating a new class would be more robust than overwriting bootstrap row class